src/Controller/TimesheetControllerTrait.php

```diff
@@ -158,6 +160,16 @@ protected function create(Request $request, $redirectRoute, $renderTemplate)
             }
         }
 
+        if ($request->query->get('project')) {
```

Comment:
I would say same here, but I am not sure if this would lead to code duplication?!?